app/tests/test_db.py

```diff
@@ -1,7 +1,4 @@
 import pytest
-from sqlalchemy import create_engine
-from sqlalchemy import exc
-from sqlalchemy.orm import sessionmaker
```

Comment:
WHY??
```suggestion
from sqlalchemy import create_engine
from sqlalchemy import exc
from sqlalchemy.orm import sessionmaker
```

Reply:
I have resolved your suggestion, sir. I hope you are fine. Have a nice day,